src/main/resources/com/google/api/codegen/csharp/gapic_client.snip

```diff
@@ -526,6 +546,52 @@
 @end
 
 # api: StaticLangApiView, method: StaticLangApiMethodView, isImp: boolean
+@private streamingRequestObjectMethodServer(api, method, isImpl)
+    @if isImpl
+        public override {@method.name}Stream {@method.name}(
```

Comment:
Do we not need an async method here as well? Unlike the bidi streaming, I'd expect this to actually make the request.